mne/preprocessing/nirs/__init__.py

```diff
@@ -11,3 +11,4 @@
 from ._optical_density import optical_density
 from ._beer_lambert_law import beer_lambert_law
 from ._scalp_coupling_index import scalp_coupling_index
+from .temporal_derivative_distribution_repair import temporal_derivative_distribution_repair
```

Comment:
should probably rename it `._temporal...` with a leading underscore. It's private and name collisions with the actual function after you import it here can become tricky later if you want to import something else from `.temporal_...`.